src/effects/native/flangereffect.cpp

```diff
@@ -4,9 +4,9 @@
 
 #include "util/math.h"
 
-const unsigned int kMaxDelay = 5000;
-const unsigned int kLfoAmplitude = 240;
-const unsigned int kAverageDelayLength = 250;
+const unsigned int kMaxDelay = 441000000;
```

Comment:
What is the unit of these values? What does they mean? A comment could be useful.